### ISSUE TITLE
Validation errors in Generate Code Snippet function

### ISSUE THREAD

**Issue by cnanney** [a85/POSTMan-Chrome-Extension#1226]
I was getting an error when trying to generate certain code snippets, but not with others. In the console I saw a `ValidationError` with the following data:

```
field: "data.url"
message: "must be uri format"
value: " https://totallyvalidurl.com"
```

I then realized I had a space at the beginning of the URL in the builder, which didn't cause any other problems until I tried to generate code. Maybe trim whitespace from beginning and end of the URL before running the generate code functions?

**Comment by abhijitkane** [comment]
Will fix this in the next release.

**Comment by abhijitkane** [comment]
Fixed in v3.0.4 - @cnanney can you confirm?

**Comment by cnanney** [comment]
Yup, fixed in v3.0.4.
